L/libclangex/build_tarballs.jl

```diff
@@ -8,34 +8,40 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "llvm.jl"))
 
 name = "libclangex"
-version = v"0.1.8"
+repo = "https://github.com/Gnimuc/ClangCompiler.jl.git"
+version = v"0.2.0"
 
-llvm_versions = [#=v"11.0.1",=# v"12.0.1", v"13.0.1", v"14.0.5"]
+llvm_versions = [v"18.1.7"]
```

Comment:
Your Julia compat is still 1.7 below.

- 1.10 is LLVM v15
- 1.11 is LLVM v16
- 1.12 is LLVM v18